app/assets/javascripts/discourse/controllers/preferences/users.js.es6

```diff
@@ -6,6 +6,9 @@ import User from "discourse/models/user";
 export default Ember.Controller.extend(PreferencesTabController, {
   saveAttrNames: ["muted_usernames", "ignored_usernames"],
   ignoredUsernames: Ember.computed.alias("model.ignored_usernames"),
+  userIsMemberOrAbove: Ember.computed.gte("model.trust_level", 2),
+  userIsStaff: Ember.computed.bool("model.staff"),
```

Comment:
I don't think this line is needed anymore?

Reply:
Yes, I was hoping to make it clearer, but sure 👍